tests/integration/Owncloud/OcisPhpSdk/OcisPhpSdkTestCase.php

```diff
@@ -44,6 +44,9 @@ public function tearDown(): void
             $drive->disable();
             $drive->delete();
         }
+        foreach($ocis->getGroups() as $group) {
+            $ocis->deleteGroup($group);
+        }
```

Comment:
I think we should remember the groups we have created in the tests and only delete those, not all. Just in case s.o. runs the test on a system that has already groups